ctl/internal/cmd/rst/pushpull.go

```
@@ -55,6 +57,7 @@ WARNING: Files are always uploaded and existing files overwritten unless the rem
 	cmd.Flags().BoolVarP(&frontendCfg.verbose, "verbose", "v", false, "Print additional details about each job (use --debug) to also print work requests and results.")
 	cmd.Flags().IntVar(&frontendCfg.width, "column-width", 35, "Set the maximum width of some columns before they overflow.")
 	cmd.Flags().BoolVarP(&backendCfg.StubLocal, "stub-local", "s", false, "Replace with a stub after the file is uploaded.")
+	cmd.Flags().BoolVar(backendCfg.Update, "update", false, "Force update the file's persistent RST configuration. Must be used with --remote-target.")
```

I think `force` might give the wrong impression since it's not dangerous or ignoring errors. Also, `persistent` is true but is inherent in the update. Maybe, `Apply the specified remote target to the file. Requires --remote-target.` or `Set remote target on the file. Requires --remote-target`

Also, add the conditional to the args function on line 40 to fail when `backendCfg.Update` is true but `backendCfg.RemoteStorageTarget` is not valid.

Do this for the pull command too

---

Updated.